config/firefox-panel.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
It'd be great to have these inherit so that we don't duplicate the config across everything. A base config would work everywhere.